SpatialGDK/Source/SpatialGDK/Public/Utils/SchemaUtils.h

```diff
@@ -211,4 +211,30 @@ inline FVector GetVectorFromSchema(Schema_Object* Object, Schema_FieldId Id)
 // Does not clear OutPath first.
 void GetFullPathFromUnrealObjectReference(const FUnrealObjectRef& ObjectRef, FString& OutPath);
 
+template <class TComponent>
+Worker_ComponentUpdate CreateComponentUpdateHelper(const TComponent& Component)
+{
+	Worker_ComponentUpdate Update = {};
```

Comment:
```suggestion
	Worker_ComponentUpdate Update{};
```

Reply:
I actually somewhat prefer the equals for structs and it's what's done through the spatial view code and everything that touches schema.

Reply:
However I think that we shouldn't be useing Worker_ComponentUpdate anywhere, so if we could change this to ComponentUpdate and use a regular constructor that would be better.

Reply:
If we need it to be like this to deal with existing code that has dubious ownership flow, then I'd argue we shouldn't be trying to make generalised tools like this. Especially as component ID could just be passed in for basically no loss of usability. As we will be removing all instances of this as soon as we can anyway.